rpmbuild/copr_rpmbuild/helpers.py

```diff
@@ -469,3 +469,15 @@ def package_version(name):
             return pkg_resources.require(name)[0].version
         except pkg_resources.DistributionNotFound:
             return "git"
+
+
+def mock_snippets_for_tags(job_tag_to_mock_snippets, tags):
+    """
+    Return a list of mock snippets for a given list of tags
+    """
+    mock_snippets = []
+    for tag in tags:
+        snippet = job_tag_to_mock_snippets.get(tag)
+        if snippet:
+            mock_snippets.append(snippet)
```

Comment:
Shouldn't we use something like this?

```python
mock_snippet = ""
for tags_to_snippet in job_tags_to_mock_snippet:
  configured_tagset = tags_to_snippet["tags"]
  mock_snippet = tags_to_snippet["mock_snippet"]
  if configured_tagset.issubset(tags):
    mock_snippet += mock_snippet + "\n"
return mock_snippet
```

This way you may configure different snippet for `x86_64+on_demand` and
for `aarch64+on_demand`.

Reply:
I think I'd prefer to map "tag sets -> snippets" (while now it is for free, it might be a complicated move in the future).  But I don't really insist.

Reply:
I still don't see how the final conf file should look like then? For me this way it's somehow counter-intuitive to have so much complicated yaml file with "tags" and "mock_snippet" fields - it's 3x times nested just to add one line to config file which sound too complicated to me.